### PR TITLE
#08 Feat: Header Section

### PR DESCRIPTION
**Pull request by mercihabam**
### Task Description _(❗️Required)_

Created the header section & layout component, then included the header inside the layout component and applied the layout for the whole project.

### Type of Task

- [ ] fix
- [ ] chore
- [x] feat
- [ ] refactor
- [ ] perf
- [ ] docs

### Guidelines

- [x] Does your code follow ES6+ syntax?
- [x] Did you use arrow functional components?
- [x] Did you thoroughly review your code?
- [x] Have you checked if your code does not introduce conflicts in the main codebase?

### Ticket Link _(❗️Required)_

https://www.notion.so/apeunit/Header-Section-6f619c34c8094e3f925a100ce6d434b3

### CODE REVIEW

File: components/Layout/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
```
Comment (Lily-Lilian) [inline components/Layout/Header.jsx]: In the newest versions of React, you don't need to import React ,maybe you can remove this import.

File: components/Layout/index.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```
Comment (Lily-Lilian) [inline components/Layout/index.jsx]: Same here!

File: components/Layout/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+import Wrapper from '../sections/Wrapper'
+
+const Header = () => {
+  return (
+    <Wrapper>
+        <div className='py-12 flex-col lg:flex-row gap-5 flex text-2xl justify-between'>
+            <div className="">Ape Unit</div>
+            <div className="max-w-201.5">Unit➇ is a pioneering technology company specialing in decentralised technologies that creates end-to-end digital experiences for protocols including <a href="#" className='text-purple'>Ethereum</a>, <a href="#" className='text-purple'>Tezos</a>, <a href="#" className='text-purple'>Near</a>, <a href="#" className='text-purple'>Algorand</a>, <a href="#" className='text-purple'>Celo</a> and more. Its team of 2,500+ digital specialists across 30+ locations on 5 continents delivers pioneering work on a global scale with a boutique culture. </div>
```
Comment (Odile078) [inline components/Layout/Header.jsx]: Instead of using the anchor tag, use the Link component from Next js

**Review by Lily-Lilian — CHANGES_REQUESTED** [review]

File: components/Layout/Header.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Wrapper from '../sections/Wrapper'
+
+const Header = () => {
+  return (
+    <Wrapper>
+        <div className='py-12 flex-col lg:flex-row gap-5 flex text-2xl justify-between'>
```
Comment (Odile078) [inline components/Layout/Header.jsx]: You forgot to add the font size for the small screens

Comment (Odile078) [inline components/Layout/Header.jsx:7]: You forgot to add the custom line-height.

**Review by Odile078 — COMMENTED** [review]
Good job but there are few changes you need to add.

**Review by alain-kubwayo — COMMENTED** [review]
Good job! You should apply the changes requested by @Odile078 and @Lily-Lilian

File: components/Layout/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+            <div className="">Ape Unit</div>
```
Comment (clever-cr) [inline components/Layout/Header.jsx]: You should remove this  class if you don't use it

**Review by clever-cr — CHANGES_REQUESTED** [review]

**Review by alain-kubwayo — CHANGES_REQUESTED** [review]
Good job. 
``` 
You should first resolve the conflicts in the LinkItem.jsx component

**Review by alain-kubwayo — DISMISSED** [review]
Well-done! 🚀

**Review by clever-cr — DISMISSED** [review]
LGTM!

**Review by Odile078 — DISMISSED** [review]

**Review by Odile078 — APPROVED** [review]

**Review by Lily-Lilian — APPROVED** [review]

**Review by alain-kubwayo — APPROVED** [review]
Great 🚀

**Pull request merged**